src/password-store.sh

```diff
@@ -588,6 +586,8 @@ cmd_git() {
 		git_add_file .gitattributes "Configure git repository for gpg file diff."
 		git config --local diff.gpg.binary true
 		git config --local diff.gpg.textconv "$GPG -d ${GPG_OPTS[*]}"
+	elif [[ $1 == "submodule" ]]; then
+		git -C "$GIT_WORK_TREE" "$@"
```

Comment:
Where is `$GIT_WORK_TREE` set to anything special, and how does that differ from the next case?